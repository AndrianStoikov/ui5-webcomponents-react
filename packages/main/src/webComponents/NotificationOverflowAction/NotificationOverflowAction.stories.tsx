import { boolean, select, text } from '@storybook/addon-knobs';
import { ButtonDesign } from '@ui5/webcomponents-react/lib/ButtonDesign';
import { NotificationOverflowAction } from '@ui5/webcomponents-react/lib/NotificationOverflowAction';
import React from 'react';

export default {
  title: 'UI5 Web Components / NotificationOverflowAction',
  component: NotificationOverflowAction
};

export const generatedDefaultStory = () => (
  <NotificationOverflowAction
    design={select('design', ButtonDesign, ButtonDesign.Transparent)}
    disabled={boolean('disabled', false)}
    icon={text('icon', '')}
    text={text('text', '')}
  />
);

generatedDefaultStory.story = {
  name: 'Generated default story'
};
