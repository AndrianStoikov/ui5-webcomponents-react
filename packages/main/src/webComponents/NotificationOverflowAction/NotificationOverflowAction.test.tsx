import { NotificationOverflowAction } from '@ui5/webcomponents-react/lib/NotificationOverflowAction';
import { mount } from 'enzyme';
import React from 'react';

describe('NotificationOverflowAction', () => {
  test('Basic Test (generated)', () => {
    const wrapper = mount(<NotificationOverflowAction />);
    expect(wrapper.render()).toMatchSnapshot();
  });
});
