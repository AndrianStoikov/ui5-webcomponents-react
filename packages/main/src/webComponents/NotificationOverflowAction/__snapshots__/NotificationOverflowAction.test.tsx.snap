// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationOverflowAction Basic Test (generated) 1`] = `
<ui5-notification-overflow-action
  design="Transparent"
/>
`;
